test-apps/ui-test-app/src/frontend/appui/widgets/demodataproviders/demoTableDataProvider.ts

```diff
@@ -195,7 +195,7 @@ export const onDropTargetDrop = (args: DropTargetArguments<DemoTableDragDropType
     const dragRow: RowItem = {
       key: id,
       extendedData: {
-        id, label, icon, type, description, parentId, dataProvider, ...rest,
+        label, icon, type, description, parentId, dataProvider, ...rest, id,

```

Comment:
@bsteinbk is it safe to change the order here?  Technically `rest` contains an id property that would have overwritten `id` before...